docs/building-a-form/common-widgets.md

```diff
@@ -0,0 +1,78 @@
+# Common widgets
+
+These common widgets are included in the us-forms-system by default. Set these widgets in a config file while building your form.
+
+Some widgets are associated with particular schema types or formats. There are additional widgets available, but the us-forms-system uses definitions they're included in, rather than just the field. To use a widget, set `ui:widget` for a field to the name of the widget. Widgets are located in [/src/js/widgets](../../src/js/widgets).
+
+Some widgets are passed as a string (e.g., `'ui: widget': 'date'`), while others are passed as the component itself (e.g., `'ui:widget': CurrencyWidget`).
```

Comment:
...passed *as a reference to* the component...

@annekainicUSDS I think in most cases these components are actually functions, right? So the examples below would need to look like: `'ui:widget': CurrencyWidgetUI('Total benefit')` and somewhere above there you would need `import CurrencyWidgetUI from 'us-forms-system/lib/js/definitions/currency'` to define the function.

Reply:
I guess what I'm seeing on forms like the [vets.gov 1990](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/1990/config/form.js) is that they aren't referencing any widgets at all directly, although they do mention `ui:widget` for the RJSF ones like `radio`. The thing about the non-builtins is that the definition files often have several lines and reference other things like a special review widget.